src/app/header/header.component.ts

```diff
@@ -21,12 +23,17 @@ import { Router } from '@angular/router';
     ]),
   ],
 })
+
 export class HeaderComponent {
+
+  constructor(private router: Router,
+    private queryData: QueryService,
```

Comment:
It should be 
private queryService: QueryService

Reply:
done

Reply:
Did you push your changes? I still dont see the change. 😅